common/src/types/v0/store/node.rs

```diff
@@ -1,31 +1,104 @@
 //! Definition of node types that can be saved to the persistent store.
 
-use crate::types::v0::{
-    openapi::models,
-    store::{
-        definitions::{ObjectKey, StorableObject, StorableObjectType},
-        ResourceUuid,
+use crate::{
+    types::v0::{
+        openapi::models,
+        store::{
+            definitions::{ObjectKey, StorableObject, StorableObjectType},
+            ResourceUuid,
+        },
+        transport::{self, NodeId},
     },
-    transport::{self, NodeId},
+    IntoOption,
 };
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
+/// Generic labels associated with the node
```

Comment:
we've been trying to tidy up the doc comments now, start with upper case (already done here) and end them with a period `.`

Reply:
Arguably you should do that only for complete sentences, but ok fixed.